src/common/legacy_config_opts.h

```diff
@@ -315,6 +315,7 @@ OPTION(paxos_trim_min, OPT_INT)  // number of extra proposals tolerated before t
 OPTION(paxos_trim_max, OPT_INT) // max number of extra proposals to trim at a time
 OPTION(paxos_service_trim_min, OPT_INT) // minimum amount of versions to trigger a trim (0 disables it)
 OPTION(paxos_service_trim_max, OPT_INT) // maximum amount of versions to trim during a single proposal (0 disables it)
+OPTION(paxos_service_trim_max_multiplier, OPT_INT) // factor by which paxos_service_trim_max will be multiplied, stops paxos_service_trim_max from dynamically changing (0 disables it)
```

Comment:
these descriptions should be moved to the set_description() calls in options.cc - new options don't need to be added to this legacy file unless they're in a performance critical path (this isn't one)